packages/components/form/src/form-field.stories.ts

```diff
@@ -81,6 +82,18 @@ export const RadioGroup: Story = {
   }
 };
 
+export const Select: Story = {
```

Comment:
it looks like that `invalid` state needs to look differently comparing to Figma:
<img width="1412" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/36182e50-d6b9-4b5c-a1c4-98d0a7dcd898">

And currently in the Storybook
<img width="789" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/e278bdc3-72f2-4383-b78e-c0f1107a52e9">

Reply:
Fixed!